fix script name for larger patch in distfiles, as there were some changes between
then and now that prevent the patch from applying

--- a/src/constants.vala.in
+++ b/src/constants.vala.in
@@ -167,8 +167,7 @@
 	public const string HTML_SCRIPT_RESOURCE_LOCATION = "resource:///" + 
 							RESOURCE_PREFIX + BOOKWORM_HTML_SCRIPT;
 
-	public const string MOBIUNPACK_SCRIPT_LOCATION = INSTALL_MOBILIB_DIR +
-							"/mobi_unpack.py";
+	public const string MOBIUNPACK_SCRIPT_LOCATION = INSTALL_MOBILIB_DIR + "/kindleunpack.py";
 
 	//Set resource paths for images and icons
 	public const string PREV_PAGE_ICON_IMAGE_LOCATION = RESOURCE_PREFIX+"16-bookworm-go-previous.svg";
